### PR TITLE
Fix emission constraint and verify results

### PR DESCRIPTION
**Pull request by jnnr**
The example add_constraints.py shows how to set an emission constraint and an inflow share rule. 

However, the emission constraint is set onto the flows, not the emissions. The emissions are the flows multiplied by the emission factor. 

This PR corrects the mistake and prints the resulting emissions to verify correctness.

### CODE REVIEW

**Comment by uvchik** [comment]
Actually this example shows how to add your own constraint. We should mention that the emission constraint already is part of solph but that it is still a good example to show how to add a not existing constraint.

**Comment by jnnr** [comment]
Thanks for the remark. I added a sentence. Apart from this, do you have any feedback?

**Comment by jnnr** [comment]
Oh, this is still v0.2. Has been idle for a while!

**Comment by uvchik** [comment]
Okay, I moved the example to v0.3 and restored the old v0.2 example. We do not need to update the v0.2 example in my opinion because nobody will look there.

Would be great to show the difference between a emission limit of 75 and 80 in a plot, but as this is an example to show how to add a constraint and not how to use the emission limit it might not be necessary. 

I added one sentences to explain it. For me the unbalanced bus is confusing as we normally do not use it. Furthermore, I added you mail address.

File: examples/oemof_0.2/flexible_modelling/add_constraints.py [hunk 1]
```diff
@@ -21,6 +21,7 @@
     pip install oemof
 
 31.10.2016 - simon.hilpert@uni-flensburg.de
+25.02.2010 - jann.launer@rl-institut.de
```
Comment (jnnr) [inline examples/oemof_0.2/flexible_modelling/add_constraints.py]:
```suggestion
25.02.2020 - jann.launer@rl-institut.de
```

**Comment by jnnr** [comment]
Cool, thanks!

File: examples/oemof_0.2/flexible_modelling/add_constraints.py (hunk 1 above)
Comment (uvchik) [reply examples/oemof_0.2/flexible_modelling/add_constraints.py]: :+1: :smile:

**Pull request merged**